src/sparseml/pytorch/sparsification/distillation/modifier_distillation.py

```diff
@@ -388,7 +387,7 @@ def _kldiv_output_loss(self, student_outputs, teacher_outputs):
             distill_head_output_losses.append(
                 self._calc_distill_head_output_loss(student_outputs, teacher_outputs)
             )
-        elif isinstance(student_outputs, Mapping):
+        elif isinstance(student_outputs, Dict):
```

Comment:
What's the motivation for restricting the type accepted here?

Reply:
`Mapping` was moved to `from collections.abc import Mapping` in python 3.10. Wasn't sure if we wanted to do weird import stuff depending on python version or just change this check.

Do you know what other types are handled by this elif branch?

Reply:
Actually looking more into it apparnetly it was deprecated in 3.3, so I think I can actually just change the import and it should be fine